eng/scripts/Language-Settings.ps1

```diff
@@ -139,15 +139,32 @@ function Get-rust-AdditionalValidationPackagesFromPackageSet ($packagesWithChang
   return $additionalPackages ?? @()
 }
 
+# $GetPackageInfoFromPackageFileFn = "Get-${Language}-PackageInfoFromPackageFile"
```

Comment:
Nit: probably just remove. It's source control so never gone. Where it is in the source file, this is just confusing.

Reply:
The `Get-${Language}-PackageInfoFromPackageFile` function is defined with a different name in every language repo but the variable used to refer to that function is the same (`$GetPackageInfoFromPackageFileFn`). This comment is a standard way of making function invocations easy to search from the declaration of the function. 

With this comment I can search for `$GetPackageINfoFromPackageFileFn` instead of having to go to `eng/common/scripts/common.ps1`, then look up `Get-${Language}-PackageInfoFromPackageFile` then find and search the variable name.